Modules/_tkinter.c

```diff
@@ -143,7 +143,8 @@ _get_tcl_lib_path(void)
         struct stat stat_buf;
         int stat_return_value;
 
-        PyObject *prefix = PySys_GetObject("prefix");  // borrowed reference
+        /* gh-125235: Should not use "prefix" which might point to venv root. */
+        PyObject *prefix = PySys_GetObject("base_prefix");  // borrowed reference
```

Comment:
I think removing `//bordered reference` is more appropriate, because you have already explained it above.

Reply:
I removed my extra comment line, since it's neither a workaround nor a corner case.

Reply:
So deprecated `Py_GetPath()` should be replaced with `PySys_GetObject("base_prefix")`, instead of `PySys_GetObject("prefix")`?